docs/documentation/release_notes/topics/26_0_0.adoc

```diff
@@ -50,6 +50,14 @@ The `java-keystore` key provider, which allows loading a realm key from an exter
 
 For more information about this subject, see link:{adminguide_link}#realm_keys[Configuring realm keys].
 
+= Customizable footer in login Theme
```

Comment:
Can we also add this information to the theming documentation?

Reply:
Yes!

Reply:
Just updated the docs, please check :)